processing/data_collection/gazette/spiders/sc_lages.py

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
Extra line. You can remove it.

Reply:
> Extra line. You can remove it.

Worry not. Just let Black do the job ; )